src/main/java/it/gov/pagopa/payment/service/qrcode/QRCodeAuthPaymentServiceImpl.java

```diff
@@ -65,4 +70,14 @@ public AuthPaymentDTO authPayment(String userId, String trxCode) {
     return authPaymentDTO;
   }
 
+  private void sendAuthPaymentNotification(TransactionInProgress trx, AuthPaymentDTO authPaymentDTO) {
+    //todo mettere if (cercare su reward-calc)
+    try {
+      log.info("[SEND_NOTIFICATION] Sending event to Notification");
+      authorizationNotificationProducer.sendNotification(trx, authPaymentDTO);
```

Comment:
if we want to handle the error, we should also check if this method return false in order to handle it